samples/python/tutorial_code/video/optical_flow/optical_flow_dense.py

```diff
@@ -2,22 +2,27 @@
 import cv2 as cv
 cap = cv.VideoCapture(cv.samples.findFile("vtest.avi"))
 ret, frame1 = cap.read()
-prvs = cv.cvtColor(frame1,cv.COLOR_BGR2GRAY)
+frame1_gray = cv.cvtColor(frame1, cv.COLOR_BGR2GRAY)
 hsv = np.zeros_like(frame1)
-hsv[...,1] = 255
+hsv[..., 1] = 255
 while(1):
     ret, frame2 = cap.read()
-    next = cv.cvtColor(frame2,cv.COLOR_BGR2GRAY)
-    flow = cv.calcOpticalFlowFarneback(prvs,next, None, 0.5, 3, 15, 3, 5, 1.2, 0)
-    mag, ang = cv.cartToPolar(flow[...,0], flow[...,1])
-    hsv[...,0] = ang*180/np.pi/2
-    hsv[...,2] = cv.normalize(mag,None,0,255,cv.NORM_MINMAX)
-    bgr = cv.cvtColor(hsv,cv.COLOR_HSV2BGR)
-    cv.imshow('frame2',bgr)
-    k = cv.waitKey(30) & 0xff
-    if k == 27:
+    if ret:
+        frame2_gray = cv.cvtColor(frame2, cv.COLOR_BGR2GRAY)
+        flow = cv.calcOpticalFlowFarneback(frame1_gray, frame2_gray, None, 0.5, 3, 15, 3, 5, 1.2, 0)
```

Comment:
> prvs,next
> frame1_gray, frame2_gray

Please don't rename vars (this introduces unnecessary changes in the patch)

Reply:
got it, i change it right now